src/server/conn_context.cc

```diff
@@ -79,6 +80,7 @@ const CommandId* StoredCmd::Cid() const {
 ConnectionContext::ConnectionContext(const ConnectionContext* owner, Transaction* tx,
                                      facade::CapturingReplyBuilder* crb)
     : facade::ConnectionContext(nullptr, nullptr), transaction{tx} {
+  acl_commands = std::vector<uint64_t>(acl::NumberOfFamilies(), acl::ALL_COMMANDS);
```

Comment:
Just a thought for the future, maybe we can invent smth more advanced than a plain vector. For example, maybe 99% of users just use ALL command or all command except a few, or except a few families. So we can encode the information in 99% cases more compact